### PR TITLE
chore: adding tNEAR for near-testnet

### PR DESCRIPTION
**Pull request by bertux**
## Description of the changes
need new native token declared on new chain to be able to test the integration in currency-api and then from end to end

### CODE REVIEW

File: packages/currency/src/native.ts [hunk 1]
```diff
@@ -57,12 +57,6 @@ export const nativeCurrencies: Record<NativeCurrencyType, (NativeCurrency & { na
       name: 'Fantom',
       network: 'fantom',
     },
-    {
-      symbol: 'BNB',
-      decimals: 18,
-      name: 'BNB',
-      network: 'bsctest',
-    },
```
Comment (alexandre-abrioux) [inline packages/currency/src/native.ts:65]: I don't understand this change

**Review by alexandre-abrioux — COMMENTED** [review]
Why the need to rename this, what's wrong with `NEAR-testnet`? I can't find any documentation on this new `tNEAR` symbol online, that's why I'm asking this.

**Review by KolevDarko — APPROVED** [review]

File: packages/currency/src/native.ts (hunk 1 above)
Comment (bertux) [reply packages/currency/src/native.ts:65]: it's just cleaning because it's not used.

**Review by alexandre-abrioux — APPROVED** [review]
Saw in DM: it's to remove confusion between the new near-testnet chain (renamed from aurora-testnet) and the NEAR-testnet currency.

**Comment by bertux** [comment]
superseded by https://github.com/RequestNetwork/currency-api/pull/236

**Pull request merged**